docs/GraFx-Developers/connectors/authorization-for-connectors/index.md

```diff
@@ -0,0 +1,287 @@
+# Authorization for Connectors
+
+Connectors support various types of authorization to enable secure integration with external systems. This document outlines the process of adding authorization to a Connector and explains the different authorization types available.
+
+## Understanding Authorization
+
+When developing your Connector's JavaScript logic, you'll often need to make requests to external APIs. While the `runtime.fetch` method makes this process straightforward, handling API endpoints that require authorization or authentication can be challenging.
+
+Hardcoding tokens directly into the Connector code is considered bad practice, as this code may be accessible to other administrators in your environment. To address this security concern, the Connector CLI tool provides four different authorization types:
+
+- Static Header Key
+- OAuth Client Credentials
+- OAuth Authorization Code
+- OAuth Resource Owner Password (Password Grant)
+
+When a user first accesses a Connector and the code makes a `runtime.fetch` call, the GraFx Platform automatically runs the required authentication workflow. The authentication result is then cached and added to subsequent requests automatically.
+
+## Authorization Limitations
+
+While the Connector framework supports the most common authorization schemas, it's important to note that completely customizable authorization is not possible. If your specific authorization requirements don't align with the four provided types, you may need to develop a workaround solution.
+
+For example, services like GraFx Media and GraFx Platform don't meet the requirements of any of the supported authorization types. In such cases, you would need to implement an intermediary service that adapts the authorization workflow to work within the Connector framework.
+
+## Define Supported Authorization Types
+Before implementing authorization, you must define the supported authorization types in your `package.json` file. This ensures consistency between your CLI commands and your package configuration.
+
+To do so, add or modify the `supportedAuth` property you `package.json` file with the types of authentication your Connector supports.
```

Comment:
Grammar, change to "To do so, add or modify the `supportedAuth` property in your `package.json` file with the types of authentication your Connector supports."